src/interface/Activity.ts

```diff
@@ -0,0 +1,5 @@
+interface Activity {
```

Comment:
I don't recommend to make a specific interface if they're not used in multiple places